app/package.json

```diff
@@ -3,46 +3,63 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "build": "next build",
-    "dev": "next dev",
+    "build": "NEXT_TELEMETRY_DISABLED=1 next build",
+    "dev": "NEXT_TELEMETRY_DISABLED=1 next dev",
```

Comment:
nit: do you need `NEXT_TELEMETRY_DISABLED` in all of these scripts since you set `disableTelemetry` in the config file?

Reply:
Hmm I assumed that I set `disableTelemetry` specifically for storybook and that these other flags are for next.js?

Reply:
Ah oops, I got the two mixed up